spec/support/app_templates/testapp/app/controllers/home_controller.rb

```diff
@@ -0,0 +1,5 @@
+class HomeController < ApplicationController
+  def show
+    render text: '', layout: 'application'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.